packages/site/src/web/index.js

```diff
@@ -31,7 +31,8 @@ require('react-live/react-live.css');
 // Enable Glamor simulate helper
 simulations(true);
 
-// Github 404 page hack to support SPA hosting
```

Comment:
Continuing on, is the `site/public/CNAME` file still needed?  Does netlify use that too?